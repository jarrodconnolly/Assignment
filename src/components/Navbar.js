import React from 'react';

// simple function navbar component for now
const NavBar = () => {
  return (
    <nav className="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0">
      <a className="navbar-brand col-sm-3 col-md-2 mr-0">Assignment</a>
      <ul className="navbar-nav px-3">
        <li className="nav-item text-nowrap">
          <a className="nav-link">Sign out</a>
        </li>
      </ul>
    </nav>
  );
}

export default NavBar;
