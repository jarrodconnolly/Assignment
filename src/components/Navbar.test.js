import React from 'react';
import ReactDOM from 'react-dom';
import Navbar from './Navbar';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Navbar />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('render matches snapshot', () => {
  const tree = renderer
    .create(<Navbar />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
