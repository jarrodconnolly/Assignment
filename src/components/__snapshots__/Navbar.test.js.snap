// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render matches snapshot 1`] = `
<nav
  className="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0"
>
  <a
    className="navbar-brand col-sm-3 col-md-2 mr-0"
  >
    Assignment
  </a>
  <ul
    className="navbar-nav px-3"
  >
    <li
      className="nav-item text-nowrap"
    >
      <a
        className="nav-link"
      >
        Sign out
      </a>
    </li>
  </ul>
</nav>
`;
